src/components/atoms/Price/Conversion.tsx

```diff
@@ -59,7 +61,8 @@ export default function Conversion({
       className={styleClasses}
       title="Approximation based on current OCEAN spot price on Coingecko"
     >
-      ≈ <strong dangerouslySetInnerHTML={{ __html: priceConverted }} />{' '}
+      {hideApproximationSign ? <> </> : <> ≈ </>}
```

Comment:
I think this would also work, right? Just a bit shorter and no need for those fragments:

```jsx
{!hideApproximationSign && '≈ ' }
```

Reply:
yup, it's a much better approach. thanks!